nms/app/packages/magmalte/app/views/subscriber/SubscriberOverview.js

```diff
@@ -61,17 +62,97 @@ const useStyles = makeStyles(theme => ({
   },
 }));
 
+type InitSubscriberStateProps = {
+  networkId: network_id,
+  setSubscriberMap: ({[string]: subscriber}) => void,
+  setSubscriberMetrics: ({[string]: Metrics}) => void,
+  enqueueSnackbar: (msg: string, cfg: {}) => ?(string | number),
+};
+
+async function InitSubscriberState(props: InitSubscriberStateProps) {
+  const {
+    networkId,
+    setSubscriberMap,
+    setSubscriberMetrics,
+    enqueueSnackbar,
+  } = props;
+  let subscribers = {};
+  const subscriberMetrics = {};
+
+  try {
+    subscribers = await MagmaV1API.getLteByNetworkIdSubscribers({networkId});
+  } catch (e) {
+    enqueueSnackbar('failed fetching subscriber information', {
+      variant: 'error',
+    });
+    return [];
+  }
+  setSubscriberMap(subscribers);
+
+  let currentUsageResp = {};
+  try {
+    currentUsageResp = await MagmaV1API.getNetworksByNetworkIdPrometheusQuery({
+      networkId: networkId,
+      query: `sum by (IMSI) (ue_traffic)`,
```

Comment:
super minor nit: I usually do `sum(<metric>) by (<label>)` since it looks a little more familiar.